rerun_cpp/src/rerun/recording_stream.hpp

```diff
@@ -167,33 +156,25 @@ namespace rerun {
         ///
         /// Logs any failure via `Error::log_on_failure`
         template <typename... Ts>
-        void log_components(const char* entity_path, const Ts&... component_array) {
-            try_log_components(entity_path, component_array...).log_on_failure();
+        void log_components(const char* entity_path, const Ts&... component_arrays) {
```

Comment:
Applies to all his little friends too
```suggestion
        void log_component_batches(const char* entity_path, const Ts&... component_batches) {